src/owner/paths/owner@orgs@delete.yaml

```diff
@@ -18,6 +18,11 @@ post:
           properties:
             data:
               $ref: '../../components/schemas/org.yaml#/org_reset_delete_in'
+            reason:
+              type: string
+              description: |
+                Reason for deleting the organisation.
+              example: 'Testing account'
```

Comment:
example can be better as we enourage not to use `testing testing` 😅

Reply:
cool, will fix this